doc/getting_started.rst

```diff
@@ -235,7 +235,41 @@ Now you can use **Lua Fun**:
     2 c
     one
 
-Please note that functions support multireturn.
+Please note that functions support multireturn. It means that some of them may
+return more than one iterator, for example functions ``partition`` and ``span``.
```

Comment:
I guess the idea of the 'functions support multireturn' sentence was about all functions: they return `gen, param, state`. A note about functions that returns two iterators is a separate thing.

Reply:
Fixed.

```asciidoc
--- a/doc/getting_started.rst
+++ b/doc/getting_started.rst
@@ -235,11 +235,13 @@ Now you can use **Lua Fun**:
     2 c
     one
 
-Please note that functions support multireturn. It means that some of them may
-return more than one iterator, for example functions ``partition`` and ``span``.
+Please note that functions support multireturn - many functions return a
+triplet ``gen``, ``param`` and ``state``, so called :ref:`iterator triplet
+<iterator_triplet>`.
 
-One call ``:unwrap()`` on generators to pass them into the ``for <..> in <...>``
-loop. Example:
+Please note that functions support returns of two iterators. Example of these
+functions are :func:`partition` and :func:`span`. One can call ``:unwrap()``
+on iterators to pass them into the ``for <..> in <...>`` loop. Example:
 
 .. code-block:: bash
 
```